Makefile

```diff
@@ -125,8 +125,11 @@ BUILDROOT/ecs-agent.tar:
 
 .generic-rpm-done:
 	./scripts/update-version.sh
+	curl -O ${AGENT_URL}
```

Comment:
It seems we shouldn't need it https://github.com/aws/amazon-ecs-init/pull/433#discussion_r724363912

Reply:
actually if we don't curl it in install we still need to manually download it, rpm build do not automatically pull resources specified in spec.